recipes/harfbuzz/all/conanfile.py

```diff
@@ -100,8 +100,8 @@ def build_requirements(self):
         self.tool_requires("meson/1.1.0")
         if not self.conf.get("tools.gnu:pkg_config", check_type=str):
             self.tool_requires("pkgconf/1.9.3")
-        if self.options.with_glib and not can_run(self):
-            self.tool_requires("glib/2.76.3")
+        if self.options.with_glib:
+            self.tool_requires("glib/<host_version>")
```

Comment:
This may require `required_conan_version = ">=1.60.0 <2.0 || >=2.0.6"` further up :D